### PR TITLE
Add breadcrumb

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/areaFlex/AreaFlexToolbarView.coffee [hunk 1]
```diff
@@ -42,10 +44,33 @@ class OpenOrchestra.AreaFlex.AreaFlexToolbarView extends OrchestraView
...
+  triggerEditArea: (event) ->
+    areaId = $(event.target).attr('data-area-id')
+    OpenOrchestra.AreaFlex.Channel.trigger 'activateEditArea', areaId if areaId?
+
+  ###*
+   * Add breadcrumb on toolbar
+   * @param {object} areaView
+  ###
+  addBreadcrumb : (areaView) ->
+    if areaView.options.area.get('area_type')? and areaView.options.area.get('area_type') != 'row'
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/areaFlex/AreaFlexToolbarView.coffee]: Can you use some kind of constant for referencing the string `row` ?

File: BackofficeBundle/Resources/public/coffee/areaFlex/AreaFlexToolbarView.coffee [hunk 2]
```diff
@@ -42,10 +44,33 @@ class OpenOrchestra.AreaFlex.AreaFlexToolbarView extends OrchestraView
...
+      $('.area-breadcrumb', @$el).prepend(
+        @renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/areaFlex/areaBreadcrumbToolbar',
+          area: areaView.options.area
+        )
+      )
+    if areaView.options.area.get('area_type')? and areaView.options.area.get('area_type') != 'root'
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/areaFlex/AreaFlexToolbarView.coffee]: same with `root`

**Comment by itkg-alavieille** [comment]
@itkg-ngilain pr updated add constant

**Comment by itkg-nanne** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**